packages/dds/sequence/src/sequence.ts

```diff
@@ -32,22 +32,18 @@ import {
 	ReferenceType,
 	SegmentGroup,
 	SlidingPreference,
-	createAnnotateRangeOp,
-	// eslint-disable-next-line import/no-deprecated
+	createAnnotateRangeOp, // eslint-disable-next-line import/no-deprecated
```

Comment:
ditto

Reply:
shouldn't prettier get these?

Reply:
#20415

Prettier allows trailing comments like:
```ts
foo, // some comment
```
I don't think it understands that eslint comments are special.